slamd/design_assistant/processing/design_assistant_service.py

```diff
@@ -12,17 +13,23 @@ def create_design_assistant_form(cls):
         design_assistant_session = DesignAssistantPersistence.get_session_for_property('design_assistant')
         progress = cls._extract_progress(design_assistant_session)
         form = DesignAssistantFactory.create_design_assistant_form()
-        if not design_assistant_session:
-            cls.init_design_assistant_session()
-            form.import_form = None
-            form.campaign_form = None
         if design_assistant_session:
-            if 'zero_shot_learner' in list(design_assistant_session.keys()):
+            if 'zero_shot_learner' in list(design_assistant_session.keys()):             
                 cls._populate_task_form_with_session_value(form, 'zero_shot_learner')
-            if 'zero_shot_learner' in list(design_assistant_session.keys()):
+                cls._populate_material_type_form_with_session_value(form, design_assistant_session, 'zero_shot_learner')
                 cls._populate_campaign_form_with_session_value(form, design_assistant_session)
-            else:
+                form.new_project_form = None
+            if 'data_creation' in list(design_assistant_session.keys()):
+                cls._populate_task_form_with_session_value(form, 'data_creation') 
+                cls._populate_material_type_form_with_session_value(form, design_assistant_session, 'data_creation')
+                cls._populate_create_project_form_with_session_value(form)
                 form.campaign_form = None
+        if not design_assistant_session:
```

Comment:
Statt 
 if not design_assistant_session
sollte hier auch einfach ein
else
funktionieren oder?